source/copilot-schema.txt

```diff
@@ -0,0 +1,63 @@
+.. _vsce-copilot-schema:
+
+===============
+/schema Command
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+The ``/schema`` command provides information about your collection
+schema. The |copilot| accesses your connected MongoDB deployment to
+provide schema information in the Copilot generated response. 
+
+Examples
+--------
+
+Provide a Sample Document
+~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Consider rewrite to not use just one nested heading.

Also, we typically require the use of numbered steps for examples now.

Reply:
Done! Because this is a UI example instead of a code example, there is only 1 step (asking the copilot chat) so I left out numbers